CMakeLists.txt

```diff
@@ -203,6 +203,8 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
   FetchContent_MakeAvailable(cutlass)
 
   list(APPEND VLLM_EXT_SRC
+    "csrc/mamba/mamba_ssm/selective_scan_fwd.cu"
+    "csrc/mamba/causal_conv1d/causal_conv1d.cu"
```

Comment:
Does Jamba work on AMD GPUs currently? Want to make sure we're not regressing there.

Reply:
I've seen ROCm support in the kernel code, but we have not tested it yet. I would say it is not support atm.